model_utilities/train/data_loader.py

```diff
@@ -19,3 +19,33 @@ def load_dataset(train_path: str, validation_path: str, test_path: str) -> ds.Da
     dataset = dataset.remove_columns("Unnamed: 0")
 
     return dataset
+
+
+def prepare_train_dataset(config: dict) -> ds.DatasetDict:
+    weights = {item[0]: float(item[1]) for item in config.items("DATASET WEIGHTS")}
+
+    paths = {key: {word: os.path.join(config["DIRECTORIES"]["preprocessed_data_dir"], f"{word}-{key}.tsv") for word in ["train", "val"]} for key in weights.keys()}
```

Comment:
Move to a helper function e.g. `get_dataset_paths`.

Reply:
```suggestion
    paths = {key: {word: os.path.join(config["DIRECTORIES"]["preprocessed_data_dir"], f"{word}-{key}.tsv") for word in ["train", "validation"]} for key in weights.keys()}
```
The term `val` can be misleading as it implies value, which it is not.

Reply:
We are generating the filenames of data to load here to avoid passing 9 different filepaths statically. We can't change this as that's how the files are named in pl-csb-data.